global.d.ts

```diff
@@ -6,7 +6,8 @@ declare interface NodeModule {
 }
 
 declare interface Window {
-  STATE: any
+  STATE: any;
+  ga: any;
```

Comment:
I have a more strict definition in [`src/missing-types.d.ts`](https://github.com/GoogleChromeLabs/squoosh/blob/master/src/missing-types.d.ts#L38)

Reply:
I've moved that to global.d.ts now to use if that's alright.